bundles/org.openhab.io.homekit/src/main/resources/OH-INF/config/config.xml

```diff
@@ -33,6 +37,10 @@
 			<description>Defines the pin, used for pairing, in the form ###-##-###.</description>
 			<default>031-45-154</default>
 		</parameter>
+		<parameter name="setupId" type="text" pattern="[0-9A-Z]{4}" required="false" groupName="core">
+			<label>setupId</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
thank you for the review. good points. fixed